.github/workflows/release.yml

```diff
@@ -35,6 +36,7 @@ jobs:
     # This mechanism ensures that Jitpack only builds a package when it is first requested and efficiently handles the generation of newly released versions.
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/checkout@v4
```

Comment:
I think this part was missing - that is why it was not working with 0.4.0 release.
As a side note: I added jitpack webhooks to repo, maybe they will work too?